conda/exceptions.py

```diff
@@ -298,9 +298,14 @@ def __init__(self, command):
         init_loggers(context)
         if command in activate_commands:
             # TODO: Point users to a page at conda-docs, which explains this context in more detail
+            # Perhaps this link would be appropriate? https://docs.conda.io/projects/conda/en/latest/commands/init.html
+            # Remind users what shell they are currently using.
+            # Can be different in job schedulers versus interactive sessions, for example
```

Comment:
```suggestion
```

It's not necessary to include these extra pieces of documentation. I believe it's pretty straightforward what's going on here.